README.md

```diff
@@ -6,13 +6,16 @@
 
 Build it:
 ```
-$ docker build -t trackerbot . 
+docker build -t trackerbot .
 ```
+
 Run it interactive:
 ```
-$ docker run -itv $(pwd):/app --name tb trackerbot
+docker run -itv $(pwd):/app --name tb trackerbot
 ```
+
 Run tests inside the container:
 ```
-$ npm run test
+npm install
```

Comment:
@christian-fei @edelprino `npm run test` it is working also without `npm install` before to me ... and I don't know why :-) wbu?